test/Microsoft.ML.Tests/NerTests.cs

```diff
@@ -36,25 +38,115 @@ public void TestSimpleNer()
                 new[] {
                 new Label { Key = "PERSON" },
                 new Label { Key = "CITY" },
-                new Label { Key = "COUNTRY"  }
+                new Label { Key = "COUNTRY"  },
+                new Label { Key = "B_WORK_OF_ART"  },
+                new Label { Key = "WORK_OF_ART"  },
+                new Label { Key = "B_NORP"  },
                 });
 
             var dataView = ML.Data.LoadFromEnumerable(
                 new List<TestSingleSentenceData>(new TestSingleSentenceData[] {
-                    new TestSingleSentenceData()
-                    {   // Testing longer than 512 words.
+                    new()
+                    {
+                        Sentence = "Alice and Bob live in the liechtenstein",
+                        Label = new string[]{"PERSON", "0", "PERSON", "0", "0", "0", "COUNTRY" }
+                    },
+                     new()
+                     {
                         Sentence = "Alice and Bob live in the USA",
                         Label = new string[]{"PERSON", "0", "PERSON", "0", "0", "0", "COUNTRY"}
+                     },
+                     new()
+                     {
+                         Sentence = "WW II Landmarks on the Great Earth of China : Eternal Memories of Taihang Mountain",
+                         Label = new string[]{"B_WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART", "WORK_OF_ART" }
+                     },
+                     new()
+                     {
+                         Sentence = "This campaign broke through the Japanese army 's blockade to reach base areas behind enemy lines , stirring up anti-Japanese spirit throughout the nation and influencing the situation of the anti-fascist war of the people worldwide .",
```

Comment:
Should the space in `army 's` be removed?

Reply:
Nope. That is correct for how the input labels are put. Trying to test edge cases with stuff like that.